tree-sitter-go/tree-sitter-go.cabal

```diff
@@ -1,21 +1,21 @@
 cabal-version:       2.2
 name:                tree-sitter-go
-version:             0.2.0.0
+version:             0.2.0.1
```

Comment:
This is a minor version bump because the major version bump is already conveyed completely by a constraint on `tree-sitter`.